shape.h

```diff
@@ -31,7 +31,7 @@ typedef uint16_t shape_id_t;
 # define SHAPE_BITMAP_SIZE 16384
 
 # define SHAPE_MAX_VARIATIONS 8
-# define SHAPE_MAX_NUM_IVS 50
+# define SHAPE_MAX_NUM_IVS 80
```

Comment:
The biggest slot size is 640, minus the 16 bytes RBasic prefix, that leaves 78 embeded ivars in the biggest  size pool.

Would it make sense to set this to 78? (assuming we want to keep this limitation)

Reply:
I'm fine with 78 (since it's >= 71).